tools/gn

```diff
@@ -404,6 +404,9 @@ def to_gn_args(args):
     if args.target_os in ['android'] and gn_args['target_cpu'] in ['x64' , 'arm64']:
       gn_args['dart_use_compressed_pointers'] = True
 
+    if args.fuchsia_target_sdk_version is not None:
```

Comment:
Why is it called fuchsia_target_sdk_version instead of fuchsia_target_api_level here?

Reply:
Because I wasn't paying close enough attention. Fixed.